string label
geometry_msgs/Pose2D pose
robotnik_msgs/ptz ptz_pose
